app/services/catalog/unshare_resource.rb

```diff
@@ -1,16 +1,12 @@
 module Catalog
   class UnshareResource
-    require 'rbac-api-client'
-    include Insights::API::Common::RBAC::Utilities
-
     def initialize(options)
       @group_uuids = SortedSet.new(options.fetch(:group_uuids, []))
       @permissions = options[:permissions]
       @object      = options[:object]
     end
 
     def process
-      validate_groups
```

Comment:
@eclarizio I don't think we can remove the validate_groups code, we need to make sure that the group ids passed in exist in the RBAC service.